int32     id
Point2d[] corners
geometry_msgs/Point     real_world_position